Added how-to for private helm repo/registries (#297)
epinio/docs: docs/howtos/customization/create_custom_service.md

For service charts the keys are proper paths into the set of fields exposed by the chart through its
`values.yaml` file. Nesting is expressed using dots (`.`). As example see `ingress.enabled` above.
:::

### Private repositories and registries

> To use charts stored in private repositories or OCI registries that requires authentication you can specify a secret where the credentials are stored.

Reviewer: ```suggestion
Specify a secret storing credentials when using charts provided by private repositories or OCI registries that require authentication.
```

@jhkrug Maybe you have something better ?

Author: Not sure about this one, let's wait for one more proposal from @jhkrug

Reviewer 2: How about this. Two shorter, active voice, more simple, sentences.

You can use charts stored in private repositories or OCI registries. You will need to use a secret that provides credentials.

Author: Sounds good to me. @andreas-kupries wdyt?

Reviewer: :+1:

Author: Thank you both, @andreas-kupries @jhkrug  done :+1:

```yaml
apiVersion: application.epinio.io/v1
kind: Service
metadata:
  name: myservice
  ...
spec:
  helmRepo:
    name: myregistry
    url: oci://ghcr.io/myregistry
    secret: oci-secret
```

> Epinio will look for the `oci-secret` in the namespace where Epinio is installed. This secret should contain a `username` and `password` fields that will be used to login into the registry or authenticate to the private repository:

Reviewer: ```suggestion
Epinio will look for the `oci-secret` in the namespace where Epinio is installed. This secret has to contain `username` and `password` fields. Their values will be used to log into the registry or authenticate to the private repository:
```

> Epinio will look for the `oci-secret` in the namespace where Epinio is installed. This secret has to contain `username` and `password` fields. Their values will be used to log into the registry or authenticate to the private repository:

Reviewer 2: ```suggestion
Epinio looks for the `oci-secret` in the namespace where it is installed. 
This secret has to contain registry, or repository, login `username` and `password` fields:
```

```
kubectl create secret generic -n epinio oci-secret --from-literal=username=myusername --from-literal=password=mypassword
```

> For example, to use the Github Container Registry a PAT with the `package:read/write` scopes is needed, and it can be used as the username and password.

Reviewer: ```suggestion
For example, to use the Github Container Registry a personal access token (`PAT`) with the `package:read/write` scopes is needed, and has to be used as both username and password.
```
